Frends.SMTP.SendEmail/Frends.SMTP.SendEmail.Tests/SendEmailTests.cs

```diff
@@ -0,0 +1,233 @@
+using Frends.SMTP.SendEmail.Definitions;
+using NUnit.Framework;
+using System;
+using System.IO;
+
+namespace Frends.SMTP.SendEmail.Tests
+{
+    /// <summary>
+    /// NOTE: To run these unit tests, you need an SMTP test server. Fill in the properties below with your values.
+    /// </summary>
+    [TestFixture]
+    public class SendEmailTests
```

Comment:
I think you could modify tests to use Docker SMTP server instead of real SMTP server: https://github.com/CommunityHiQ/Frends.Community.Email/blob/master/Frends.Community.Email.Tests/SendEmailTests.cs#L10